src/components/image/SvgImage.tsx

```diff
@@ -5,10 +5,11 @@ const SvgXml = SvgPackage?.SvgXml;
 
 export interface SvgImageProps {
   data: any; // TODO: I thought this should be string | React.ReactNode but it doesn't work properly
+  fill?: string; // fill works on files if SVGR is configured appropriately, see https://github.com/kristerkari/react-native-svg-transformer/blob/f10c8d579490826227d1bcfa84ba6eabc124b70a/README.md#changing-svg-fill-color-in-js-code
```

Comment:
1. Is `fill` always the same as `tintColor`? If yes, I'd change the name.
2. I suggest putting this comment as a doc (if we want to expose it at some point).

Reply:
The problem is, that the SVG component from react-native-svg also has a `tintColor` prop, which appears to be incompatible with color strings like `#12297a`. At least on iOS that causes an error, because in the original line 23 of this file (`return <File {...others}/>;`) all props passed to the `Image` component except for those destructured explicitely for this component are passed through.

Regarding the documentation: The SvgImage component does not have its own page in your documentation. I did however add the same info to the `@notes` in the Image component documentation